Description: make crashpad build on mips64el
 This is needed to workaround error from GNU assembler:
 crashpad_info_note.S:45: Error: PC-relative reference to a different section
 .
 crashpad will not work on mips64el properly, but it is invoked only when
 something crashed anyway.
Author: Ma Aiguo <maaiguo@uniontech.com>
Forwarded: https://codereview.qt-project.org/c/qt/qtwebengine-chromium/+/334478
Last-Update: 2021-04-06

--- a/src/3rdparty/chromium/third_party/crashpad/crashpad/client/crashpad_info_note.S
+++ b/src/3rdparty/chromium/third_party/crashpad/crashpad/client/crashpad_info_note.S
@@ -42,7 +42,12 @@ name_end:
   .balign NOTE_ALIGN
 desc:
 #if defined(__LP64__)
+#if defined(__mips__)
+  .quad CRASHPAD_INFO_SYMBOL
+#else
   .quad CRASHPAD_INFO_SYMBOL - desc
+#endif
+
 #else
   .long CRASHPAD_INFO_SYMBOL - desc
 #endif  // __LP64__
